### PR TITLE
iptables: move to usr/bin.

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: srcpkgs/iptables/template [hunk 1]
```diff
@@ -51,6 +52,5 @@ iptables-devel_package() {
 		vmove "usr/lib/*.so"
 		vmove usr/include
 		vmove usr/lib/pkgconfig
-		vmove usr/share/man/man3
```
Comment (beefcurtains) [inline srcpkgs/iptables/template]: These manpages should remain in iptables-devel.

Comment (thypon) [inline srcpkgs/iptables/template]: If only it'd inside. The build only fails if i leave that line. Try it yourself.

File: srcpkgs/iptables/template [hunk 2]
```diff
@@ -13,6 +12,7 @@ homepage="http://www.netfilter.org/"
 license="GPL-2"
 distfiles="$homepage/projects/iptables/files/$pkgname-$version.tar.bz2"
 checksum=52004c68021da9a599feed27f65defcfb22128f7da2c0531c0f75de0f479d3e0
+configure_args="--sbindir=/usr/bin"
```
Comment (beefcurtains) [inline srcpkgs/iptables/template]: This overrides the configure_args above, that's why the package broke.

Comment (thypon) [inline srcpkgs/iptables/template]: ops my fault i think

**Pull request merged**